controllers/openstackmachine_controller.go

```diff
@@ -288,8 +288,6 @@ func (r *OpenStackMachineReconciler) reconcileNormal(ctx context.Context, logger
 		return ctrl.Result{}, err
 	}
 
-	logger.Info("Creating Machine")
```

Comment:
Not good message is true. I suggest to update to 'Reconciling Machine create started'
in contrast to L370 logger.Info("Reconciled Machine create successfully").